### PR TITLE
DARIAH Winter School 2019

### PR DESCRIPTION
**Pull request by mr43**
Draft version of the DARIAH Winter School 2019

### CODE REVIEW

File: contents/events/desir_winterschool2019/index.mdx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  - dariah-eu
+  - desir
+  - fcsh-nova
+social:
+  - name: website
+    url: http://dariah.eu/
+  - name: email
+    url:
+  - name: flickr
+    url: https://www.flickr.com/photos/151019225@N04/sets/72157679126801606/
```
Comment (stefanprobst) [inline contents/events/desir_winterschool2019/index.mdx]: this link seems to point to the 2016 winterschool photos

File: contents/events/desir_winterschool2019/index.mdx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    url:
+  - name: storify
+    url:
+  - name: twitter
+    url: https://twitter.com/DARIAHeu
+  - name: googleplus
+    url:
+  - name: facebook
+    url:
+synthesis: docs/DARIAH_Winter_School-FullSynthesis.pdf
```
Comment (stefanprobst) [inline contents/events/desir_winterschool2019/index.mdx]: is this the correct file?

File: contents/events/desir_winterschool2019/session01.mdx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+Fernanda has a PhD in Contemporary History by the Faculdade de Ciências Sociais
+e Humanidades, Universidade Nova de Lisboa and is Associate Professor in the
+same faculty. Researcher and former President of the Institute for Contemporary
+History, Prof. Fernanda Rollo has coordinated research projects and has several
+texts published in the areas of Economic and Social History, History of
+Portugal, and history of innovation and organization of science. She is former
+Secretary of State for Science, Technology and Higher Education.
+
+<Youtube id="OqYJNUmhJzw" caption="Laurent Romary (Former DARIAH Director)" />
```
Comment (stefanprobst) [inline contents/events/desir_winterschool2019/session01.mdx]: is this youtube video here on purpose?

File: contents/events/desir_winterschool2019/session02.mdx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  - erzsebet-toth-czifra
+#Add relevant names as listed in the person.yaml doc.
+synthesis: docs/2.1 Toth-Czifra.pdf
+---
+
+# Template for Event Sessions
+
+## Abstract
+
+_Session 1_
```
Comment (stefanprobst) [inline contents/events/desir_winterschool2019/session02.mdx]: do we need these session markers?

File: contents/events/desir_winterschool2019/index.mdx (hunk 1 above)
Comment (mr43) [reply contents/events/desir_winterschool2019/index.mdx]: Yes, I have asked our communication officer to upload the photos on Flickr. Once ready, I will change the link accordingly.

File: contents/events/desir_winterschool2019/index.mdx (hunk 2 above)
Comment (mr43) [reply contents/events/desir_winterschool2019/index.mdx]: No, we need to prepare the full synthesis still.

File: contents/events/desir_winterschool2019/session01.mdx (hunk 3 above)
Comment (mr43) [reply contents/events/desir_winterschool2019/session01.mdx]: It was just a test, videos should be ready soon.

**Pull request merged**